org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/backend/filebased/MultiRepository.java

```diff
@@ -268,6 +457,31 @@ public void putContentToFile(RepositoryFileReference ref, InputStream inputStrea
         return result;
     }
 
+    @Override
+    public Path ref2AbsolutePath(RepositoryFileReference ref) {
+        return null;
```

Comment:
You cannot return null here. This will have an impact on the whole system.
Instead, you should search through all repositories and return the corresponding Path. This comment applies to the four following methods.